pallets/claims/src/lib.rs

```diff
@@ -105,15 +106,26 @@ pub mod pallet {
 	#[pallet::getter(fn claims)]
 	pub type Claims<T: Config> = StorageMap<_, Blake2_128Concat, EthereumAddress, BalanceOf<T>, ValueQuery>;
 
+	/// Vesting schedule for a claim.
+	/// The balance is how much should be unlocked per block.
+	/// The block number is when the vesting should start.
+	#[pallet::storage]
+	#[pallet::getter(fn vesting)]
+	pub type Vesting<T: Config> = StorageMap<_, Identity, EthereumAddress, (BalanceOf<T>, T::BlockNumber), ValueQuery>;
```

Comment:
one suggestion here would be to use OptionQuery instead of ValueQuery.

It will simplify the "check and get" in the code - you can just do one `get` and it returns an option  and it will be None if it is does not exists.